test/events.test.js

```diff
@@ -0,0 +1,251 @@
+import chai from 'chai';
+import request from 'supertest';
+import bcrypt from 'bcrypt';
+import Server from '../server';
+import mongo from '../server/common/mongo';
+import userModel from '../server/models/user';
+import eventModel from '../server/models/event';
+
+
+const expect = chai.expect;
+
+chai.use(require('chai-like'));
```

Comment:
Do we require these? I tried to run after removing them and it seemed to work well, so we can remove them if they aren't required.